distribution/src/conf/file.properties

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+stockQuoteEP=http://localhost:9000/services/SimpleStockQuoteService
```

Comment:
Let's remove this stockQuoteEP and move it to the test case configuration files.
And add comment about the default file specifying what we should put in this file

Reply:
Fixed with the commit https://github.com/wso2/product-ei/pull/4812/commits/d5b6c5c11a99aebb97d38259d0d74819b1607993